src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -2314,12 +2314,12 @@ void InterpreterSelectQuery::addPrewhereAliasActions()
 }
 
 /// Based on the query analysis, check if using a trivial count (storage or partition metadata) is possible
-std::optional<UInt64> InterpreterSelectQuery::getTrivialCount(UInt64 max_parallel_replicas)
+std::optional<UInt64> InterpreterSelectQuery::getTrivialCount(UInt64 allow_experimental_parallel_reading_from_replicas)
 {
     const Settings & settings = context->getSettingsRef();
     bool optimize_trivial_count =
         syntax_analyzer_result->optimize_trivial_count
-        && (max_parallel_replicas <= 1)
+        && (allow_experimental_parallel_reading_from_replicas <= 1)
```

Comment:
In the original PR https://github.com/ClickHouse/ClickHouse/pull/63151/files#diff-5452971ae18972aa669265008a29b3732a99d9415f5b3af10d573c0807667a22R2427

`allow_experimental_parallel_reading_from_replicas <= 1`
https://github.com/ClickHouse/ClickHouse/blob/7b19c65890741b44214d0f61746331777c0ab135/src/Interpreters/InterpreterSelectQuery.cpp#L2427

Is that an intentional change or just a typo?

Reply:
Ah yes, need to backport one more PR to this jumbo pr